README.md

```diff
@@ -5,6 +5,14 @@ as part of a research study.
 
 ## Installation instructions
 
+We have a set of easy instructions for users running Ubuntu 18.04+ (including WSL
```

Comment:
It might be a good idea at this point to have the user check to make sure sudo will work.  Depending on their luck, we can point them in the right direction.

Reply:
Added statements to install instructions.